tools/src/ExpoCLI.ts

```diff
@@ -1,3 +1,5 @@
+import { extractLocalNpmTarballAsync } from '@expo/cli/build/src/utils/npm';
```

Comment:
Why do you import it from the build folder?

Reply:
I don't think that one is directly exported up to the top, but I will check.

Reply:
Verified, this method must be imported this way, and not from just `@expo/cli`.